doc/conf.py

```diff
@@ -128,6 +128,7 @@
                             r"experiment-user-cases/.*",
                             r"rrfs_a/*",
                             r"develop-20240618/*",
+                            r"develop-20250321/.*",
```

Comment:
We probably don't need to reference any links at the `develop-20240618` location from the line above. Any links currently pointing there should be updated to point at the `develop-20250321` links.